README.md

```diff
@@ -13,7 +8,33 @@ SomaScan data deliverable. Data in this file is **not** intended for
 biological analysis purposes or to provide any metrics for SomaScan data in
 general.
 
-### ADAT File Format
+---------------------
+
+## Files 
+
+* example_data.adat
+* example_data_v4.1_plasma.adat 
+
+
+## Installation
+
+The example ADAT files in this repository can be retrieved in one of two ways:
+
+1) Cloning the repo to your local machine
+
+2) Using `wget` to retrieve individual ADAT files from the repo; see examples 
+below
+
+```
+# Retrieve just the v4.0 (5k) ADAT
```

Comment:
Remind me ... externally we are calling the versions by the `k`, rather than the SomaScan version right?
So perhaps we should re-arrange these to be "5k ADAT (v4.0)"

Reply:
Will do!